app/assets/stylesheets/administrate/components/_cells.scss

```diff
@@ -1,4 +1,6 @@
 .cell-label {
+  padding-top: $top-space;
+
```

Comment:
Can you provide an example (screenshot) of how cell labels should also receive this padding? I can't recall from memory right now 😓

Reply:
Sure

![cell-label](https://user-images.githubusercontent.com/54637230/72831534-d8804280-3c61-11ea-96cb-dc95bf6ea9f3.gif)

Reply:
Thank you, I get it now :+1: